libcxx/include/vector

```diff
@@ -401,9 +402,16 @@ public:
   typedef typename __alloc_traits::difference_type difference_type;
   typedef typename __alloc_traits::pointer pointer;
   typedef typename __alloc_traits::const_pointer const_pointer;
-  // TODO: Implement iterator bounds checking without requiring the global database.
+#ifdef _LIBCPP_ABI_BOUNDED_ITERATORS
+  // The pointer must be passed through __wrap_iter because
+  // __alloc_traits::pointer may not be detected as a continguous iterator on
+  // its own.
```

Comment:
```suggestion
  // The pointer must be passed through __wrap_iter because
  // __alloc_traits::pointer may not be detected as a continguous iterator on
  // its own before C++20.
```